stellargraph/data/explorer.py

```diff
@@ -780,3 +783,166 @@ def _check_neighbourhood_sizes(self, in_size, out_size):
             self._raise_error(
                 "The number of hops for the in and out neighbourhoods must be the same."
             )
+
+
+@experimental(reason="requires more thorough testing and documentation")
```

Comment:
Do we have any issues for these tasks? If not, could we open one or more and then

```suggestion
@experimental(reason="requires more thorough testing and documentation", issues=[...issue number here...])
```

Reply:
Thanks, I've created #827 #828 #832